CHANGELOG/CHANGELOG-3.5.md

```diff
@@ -29,7 +29,7 @@ Previous change logs can be found at [CHANGELOG-3.4](https://github.com/etcd-io/
 - Fix [panic in etcd validate secure endpoints](https://github.com/etcd-io/etcd/pull/16565)
 
 ### Dependencies
-- Compile binaries using [go 1.20.10](https://github.com/etcd-io/etcd/pull/16745).
+- Compile binaries using [go 1.20.11](https://github.com/etcd-io/etcd/pull/16915).
```

Comment:
3.5.10 has already been released. Please create a new section for 3.5.11 to hold this change.

Reply:
Good catch!

Reply:
Thank you @ahrtr. Do I only add the dependencies part in the new section for 3.5.11, and leave the other parts vacant?

Reply:
Example pr here for setting up changelog with new sections: https://github.com/etcd-io/etcd/pull/15841.

You only need to add the dependencies section initially, other sections will be added later as required.

Reply:
The new section [3.5.11](https://github.com/etcd-io/etcd/blob/main/CHANGELOG/CHANGELOG-3.5.md#v3511-tbd) has already been created in another PR, please rebase this PR, and add the item under the new section. Thanks.

Reply:
Thanks @jmhbnz & @ahrtr I have updated the PR. Let me know if it looks good now.